C++/Is_prime.cpp

```diff
@@ -0,0 +1,41 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int main(){
+    int number;
+    cin>>number;
+
+    //CHECK PRIME
+
+    // 1) O(N) complexity
+    /*
+    What we will do here is we will iterate from 2 till number -1
+    and check if the number is divisible by any of this. If it is then it is not
+    a prime number
+    */
+
+    for(int i=2;i<number;i++){
```

Comment:
Better implement this as one function

Reply:
Okay sure trying to do

Reply:
Please re-open the PR and work! 😁

Reply:
I already made a new issue and made a PR there. It would be great if you could check that !